### PR TITLE
updates node and npm. adds package-lock.json back

### PR DESCRIPTION
**Pull request by esotrope**
Updates node and npm to latest stable release.
It's a good idea to update node/npm on your system to the same versions as well
```bash
brew upgrade node
npm install -g npm@5.5.1
```
I think the package-lock.json were not being created because we started using node 4.5.0
An update to 5.5.1 seemed to do the trick.

### CODE REVIEW

File: admin-webapp/src/main/webapp/package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "@types/file-saver": "0.0.1",
     "@types/lodash": "^4.14.68",
     "angulartics2": "^2.2.2",
+    "bootstrap": "^3.3.7",
```
Comment (esotrope) [inline admin-webapp/src/main/webapp/package.json]: needed an explicit dependency installation for our `style.less` to reference stylesheets in the `node_modules/bootstrap/less` dir

**Comment by esotrope** [comment]
Any aversions to merging this?

File: admin-webapp/build.gradle [hunk 2]
```diff
@@ -51,8 +51,8 @@ dependencies {
 }
 
 node {
-    version = '7.10.0'
-    npmVersion = '4.5.0'
+    version = '8.9.1'
+    npmVersion = '5.5.1'
```
Comment (wtritch) [inline admin-webapp/build.gradle]: Might want to put out a note that devs should update their node and npm versions?

**Comment by wtritch** [comment]
Thank you for doing this!

File: admin-webapp/build.gradle (hunk 2 above)
Comment (esotrope) [reply admin-webapp/build.gradle]: it's in the description but I'll echo it here:
It's a good idea to update node/npm on your system to the same versions as well
```bash
brew upgrade node
npm install -g npm@5.5.1
```

**Pull request merged**